src/locales/cn/messages.po

```diff
@@ -0,0 +1,58 @@
+msgid ""
```

Comment:
Hello translators!

This pull request is our first attempt at internationalizating of Juicebox. We are starting with Chinese translations.

This file is where Chinese translations will live. In this file, each translation has 2 parts:

1. `msgid` - the English string.
1. `msgstr` - the Chinese translation - this is where we need your help!

### How to add translations

For each `msgstr` for this file, add a comment on the line and suggest the Chinese translation. Here is an example: https://github.com/jbx-protocol/juice-interface/pull/236/files#r760041409